cmd/push.go

```diff
@@ -76,6 +77,23 @@ func push(projectID string, projectDir string, pushTag string, openInBrowser boo
 		return err
 	}
 
+	// migrate app name from spacefile to discovery file if present
+	if s.AppName != "" {
+		shared.Logger.Printf("\n%s The %s field was recently moved from the Spacefile to the Discovery.md file.\n\n", emoji.ErrorExclamation, styles.Code("app_name"))
+
+		migrateAppName, err := confirm.Run(fmt.Sprintf("Do you want to move %s from your Spacefile to the Discovery.md file automatically? (y/n)", styles.Code("app_name")))
```

Comment:
This will break the cli in non-interactive scenarios.

You can use the IsOutputInteractive function to check for this.